### ISSUE TITLE
Failure while creating edge hostname with both IP behaviors enabled

### ISSUE THREAD

**Issue by p7s1-wolfgang-dworzak** [akamai/terraform-provider-akamai#191]
### Terraform Version
Terraform v0.13.5 (Akamai Provider 0.10.2)

### Affected Resource(s)
- akamai_edge_hostname

### Terraform Configuration Files
```hcl
terraform {
    required_providers {
        akamai = {
            source  = "akamai/akamai"
        }
    }
}

provider "akamai" {
    edgerc = "~/.edgerc"
}

data "akamai_contract" "livestreaming_contract" {
    group = var.group
}

data "akamai_group" "livestreaming_group" {
    name = var.group
    contract = data.akamai_contract.livestreaming_contract.id
}

resource "akamai_cp_code" "livestreaming_cpcode" {
    name      = var.cp_code_name
    contract  = data.akamai_contract.livestreaming_contract.id
    group     = data.akamai_group.livestreaming_group.id
    product   = var.product
}

resource "akamai_edge_hostname" "livestreaming_edge_hostname" {
    contract      = data.akamai_contract.livestreaming_contract.id
    group         = data.akamai_group.livestreaming_group.id
    product       = var.product
    edge_hostname = var.property_name
    ipv4          = true
    ipv6          = true
}
```

### Debug Output
https://gist.github.com/p7s1-wolfgang-dworzak/0e5cd40a5c2f15a7b4b9067e1ffb6e26

### Expected Behavior
Edge hostname should be created with IPv4 and IPv6 enabled.

### Actual Behavior
Edge hostname was not created at all, terraform apply exited with an error. If both IP otions are not set in the config the terraform apply created the resource akamai_edge_hostname. If afterwards both options are added to the config and the change it applied it works without an error.

### Steps to Reproduce

1. `terraform apply`

**Comment by zstlaw** [comment]
We saw this issue last week and have been working on a fix for it already.

**Comment by zstlaw** [comment]
This issue has been identified and fixed through release 1.0.0. Both ipv4, ipv6 boolean attributes have been removed and replaced with the ip_behavior argument To set both IPv4 and IPv6 behaviors, set the ip_behavior argument to IPV6_COMPLIANCE. For more details, review
https://registry.terraform.io/providers/akamai/akamai/latest/docs/resources/property_edge_hostname#ip_behavior.
